vuepress/docs/next/tutorials/create/pb/publish-simple-bundle.md

```diff
@@ -19,30 +19,29 @@ This tutorial describes how to create a simple Entando Bundle and deploy it into
 
 Bundles are generated using the `ent bundle` command and its convenience methods.
 
-## Initialize a Bundle Project
+## Create a Simple Widget
 
-1. Create the bundle structure by scaffolding default files and folders:
+1. Initialize a bundle project:
    ``` sh
    ent bundle init YOUR-BUNDLE-NAME
    ```
-## Add a Simple Widget
+   This generates the required bundle structure by scaffolding default files and folders.
 
-1. From the bundle root folder, create and enter a widget directory:
+2. Create a widget folder inside the `platform` folder of your bundle's root folder:
    ``` sh
    mkdir -p platform/widgets
    ```
-   Descriptor files are organized by component type, with all micro frontends, microservices and platform-specific components located under their respective parent folders. The parent folder "platform" contains the child folder "widgets" in which to store the widget descriptor file.
-
-2. Create a widget descriptor file within the widget directory:
+   Descriptor files are organized by component type, with all micro frontends, microservices and platform-specific components located under their respective parent folders. The parent folder `platform` and child folder `widgets` complete the path to the widget descriptor file.
+   
+3. Create a widget descriptor file inside the widget directory:
    ``` sh
    touch platform/widgets/example-widget.yaml
    ```
-   A corresponding top-level bundle descriptor file is not necessary.
-
-   Platform-specific components use descriptor file names and locations (e.g. `platform/widgets/example-widget.yaml`) which are arbitrarily and explicitly defined within the bundle descriptor (`entando.json`). Convention is to group components by type with all widgets in one directory, all page templates in another, etc.
+   Platform-specific components are assigned descriptor file names and locations by the user (e.g. `widgets/example-widget.yaml`). These must be defined at the `platform` level, where convention is to group components by type, in subfolders of the same name (e.g. `platform/widgets`).
```

Comment:
We should probably tweak this slightly. In v1, the name of the folders was by convention but because the files were linked directly from the parent descriptor they could be changed without penalty. With v5, the folder names matter and have to match the names of the components.